From ca5ec8f637300ff653a0ea9d9786c1bce3d8bf93 Mon Sep 17 00:00:00 2001
From: Mirza Krak <mirza.krak@northern.tech>
Date: Tue, 10 Jul 2018 07:38:17 +0000
Subject: [PATCH 1/1] ARM: rcar_gen3: do not error out if ARCH is not defined

This causes a problem when running:

    make env

Where some defines are not present, but config.h is included in the
build to include the default environment in fw tools.

Signed-off-by: Mirza Krak <mirza.krak@northern.tech>
---
 arch/arm/include/asm/arch-rcar_gen3/rcar_gen3.h | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/arch/arm/include/asm/arch-rcar_gen3/rcar_gen3.h b/arch/arm/include/asm/arch-rcar_gen3/rcar_gen3.h
index 54d558ca53..ca93faa630 100644
--- a/arch/arm/include/asm/arch-rcar_gen3/rcar_gen3.h
+++ b/arch/arm/include/asm/arch-rcar_gen3/rcar_gen3.h
@@ -14,11 +14,7 @@
  #include <asm/arch/r8a7795.h>
  #elif defined(CONFIG_R8A7796)
  #include <asm/arch/r8a7796.h>
- #else
- #error "SOC Name not defined"
  #endif
-#else
- #error "not support ARCH."
 #endif /* CONFIG_RCAR_GEN3 */
 
 #ifndef __ASSEMBLY__
-- 
2.11.0

